graphql/queries/userInfo.ts

```diff
@@ -0,0 +1,54 @@
+import { gql } from 'apollo-boost'
+
+const USER_INFO = gql`
+  query USER_INFO($username: String!) {
```

Comment:
let's try to stick with camelCase for the names. this name will be used by the generator when creating the types and helpers

`query userInfo { ... }`